assets/sass/components/global/_googlesitekit-zero-data.scss

```diff
@@ -0,0 +1,85 @@
+/**
+ * CTA styles.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#wpadminbar .googlesitekit-zero-data {
+	height: 100%;
+	padding: $grid-gap-phone;
+	width: 100%;
+
+	@media (min-width: $bp-desktop) {
+		padding: $grid-gap-desktop;
+	}
+
+	&--error {
+		// background-color: $c-black;
+		border: 3px solid $c-error;
+	}
+
+	.googlesitekit-adminbar-modules & {
```

Comment:
This would result in a selector like `.googlesitekit-adminbar-modules #wpadminbar .googlesitekit-zero-data` would it not? I don't think that would actually match anything because `#wpadminbar` is above `.googlesitekit-adminbar-modules`.